### ISSUE TITLE
Webpack resolver alias dosen't work for js-hyperclick?

### ISSUE THREAD

**Issue by zp1112** [AsaAyers/js-hyperclick#54]
When i have defined the alias like 

```js
resolve: {
    extensions: ['', '.js', '.vue'],
    alias: {
      vue: 'vue/dist/vue.js',
      components: path.resolve(__dirname, '../app/components')
    }
  },
```

and then use it in modules like this:

```js
import Ratings from 'components/ratings/ratings';
```

and when i use cmd+click to go-to-declaration,it throw an error which means the alias didn't work.

![image](https://cloud.githubusercontent.com/assets/14245159/22049320/7a46654c-dd6d-11e6-919c-5f3639d3f6de.png)

**Comment by AsaAyers** [comment]
You have to add a `moduleRoots` to your `package.json` as documented [here](https://github.com/AsaAyers/js-hyperclick#i-configured-babeleslintflowwebpacketc-to-avoid--in-my-imports-how-can-i-configure-js-hyperclick).

**Comment by AsaAyers** [comment]
I just published v1.15 with a new feature I'm calling "Custom Resolvers". It will allow you to add a JS file to your project that can implement your aliases. [PR and documentation](https://github.com/AsaAyers/js-hyperclick/pull/105)
